sonic-xcvrd/tests/test_y_cable_helper.py

```diff
@@ -0,0 +1,1839 @@
+from xcvrd.xcvrd_utilities.port_mapping import *
+from xcvrd.xcvrd_utilities.sfp_status_helper import *
+from xcvrd.xcvrd_utilities.y_cable_helper import *
+from xcvrd.xcvrd import *
```

Comment:
not required

Reply:
I use functions and variables from all of the imports mentioned here, and removing them broke the unit tests. So I left them alone.